sevsnpmeasure/vmsa.py

```diff
@@ -182,9 +182,10 @@ def build_save_area(eip: int, sev_features: int, vcpu_sig: int, vmm_type: VMMTyp
             xcr0=0x1,
         )
 
-    def __init__(self, sev_mode: SevMode, ap_eip: int, vcpu_sig: int, vmm_type: VMMType = VMMType.QEMU):
+    def __init__(self, sev_mode: SevMode, ap_eip: int, vcpu_sig: int, guest_features: int,
+                 vmm_type: VMMType = VMMType.QEMU):
         if sev_mode == SevMode.SEV_SNP:
-            sev_features = 0x1
+            sev_features = guest_features
         else:
             sev_features = 0x0
```

Comment:
let's remove this condition.
When calling from `seves_calc_launch_digest`, always pass 0x0 for the guest_features argument.
When calling from the SNP launch digest functions, pass the user-defined guest_features from the command-line.